source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst

```diff
@@ -76,10 +86,10 @@ If you want to cosign a transaction involving a multisig account, you should be
         :start-after:  /* start block 02 */
         :end-before: /* end block 02 */
 
-    .. viewsource:: ../../resources/examples/typescript/aggregate/CosigningAggregateBondedTransactions.js
-        :language: javascript
-        :start-after:  /* start block 02 */
-        :end-before: /* end block 02 */
+    .. viewsource:: ../../resources/examples/java/src/test/java/symbol/guides/examples/aggregate/CosigningAggregateBondedTransactions.java
```

Comment:
Everywhere else Java examples have been ADDED, except here (and below) where Java examples have REPLACED the Javascript ones. Is this correct?

Reply:
good catch pushed a fix